app/src/main/java/org/mozilla/fenix/tabstray/browser/OtherHeaderBinding.kt

```diff
@@ -19,9 +19,13 @@ class OtherHeaderBinding(
     private val showHeader: (Boolean) -> Unit
 ) : AbstractBinding<TabsTrayState>(store) {
     override suspend fun onState(flow: Flow<TabsTrayState>) {
-        flow.ifAnyChanged { arrayOf(it.normalTabs, it.searchTermGroups) }
+        flow.ifChanged { Pair(it.normalTabs, it.tabPartition) }
```

Comment:
See comment above. I am confused why we have a single partition here. Is it because we're just starting with the search one, and add inactive later?

Reply:
But even then, it would have to have a different name.

Reply:
Yes, we're only using one `TabPartition` for search term.  We will update once we add inactive later.  I will rename this to `searchTermPartition`.  Thanks